### PR TITLE
Add a parameter to control the sort order of dependencies in dependency management.

### PR DESCRIPTION
**Pull request by ssquan**
Issues: #210

### CODE REVIEW

**Comment by Ekryd** [comment]
Nice!! You can follow `sortDependencies` and `sortDependencyExclusions` to look for which tests they are involved in

File: maven-plugin/src/main/java/sortpom/AbstractParentMojo.java [hunk 1]
```diff
@@ -94,6 +94,14 @@ abstract class AbstractParentMojo extends AbstractMojo {
   @Parameter(property = "sort.sortDependencyExclusions")
   String sortDependencyExclusions;
 
+  /**
+   * Comma-separated ordered list how dependencies in dependency management should be sorted. Example:
+   * scope,groupId,artifactId. If scope is specified in the list then the scope ranking is COMPILE,
+   * PROVIDED, SYSTEM, RUNTIME, IMPORT and TEST. The list can be separated by ",;:"
+   */
```
Comment (Ekryd) [inline maven-plugin/src/main/java/sortpom/AbstractParentMojo.java:101]: You should mention something about fallback here

File: sorter/src/test/java/sortpom/wrapper/CustomSortOrderFileTest.java [hunk 2]
```diff
@@ -37,6 +37,7 @@ private String getToStringOnCustomSortOrderFile()
             .setSortEntities(
                 "scope,groupId,artifactId",
                 "groupId,artifactId",
+                    "scope,groupId,artifactId",
```
Comment (Ekryd) [inline sorter/src/test/java/sortpom/wrapper/CustomSortOrderFileTest.java:40]: Shouldn't the `fmt-maven-plugin`take care of this indentation?

**Comment by Ekryd** [comment]
Hi! I have some busy days ahead right now, but hope to review this soon. Thanks for your contribution

**Comment by ssquan** [comment]
> Hi! I have some busy days ahead right now, but hope to review this soon. Thanks for your contribution
Never mind.

File: maven-plugin/src/it/mojo-description/expected.log [hunk 3]
```diff
@@ -116,6 +116,15 @@ sortpom:sort
       Comma-separated ordered list how exclusions, for dependencies, should be
       sorted. Example: groupId,artifactId The list can be separated by ',;:'
 
+    sortDependencyManagement
+      User property: sort.sortDependencyManagement
+      Comma-separated ordered list how dependencies in dependency management
+      should be sorted. Example: scope,groupId,artifactId. If scope is
+      specified in the list then the scope ranking is COMPILE, PROVIDED,
+      SYSTEM, RUNTIME, IMPORT and TEST. The list can be separated by ',;:'. It
+      would take precedence if present and would fallback sortDependencies if
```
Comment (Ekryd) [inline maven-plugin/src/it/mojo-description/expected.log:125]: fallback to sortDependencies

Comment (ssquan) [reply maven-plugin/src/it/mojo-description/expected.log]: Fixed.

**Comment by Ekryd** [comment]
This code would need an integration test (under `maven-plugin/src/it`) that ensures that the order for dependency management and dependencies can be set independently

**Comment by Ekryd** [comment]
I added a test for verifying that the parameter is properly propagated to `elementWrapperCreator` and also merged in the latest changes from master. I am using the branch [issue-210-Suggestion_Add_a_parameter_to_control_the_sort_of_dependencies_in_dependencymanagement](https://github.com/Ekryd/sortpom/tree/issue-210-Suggestion_Add_a_parameter_to_control_the_sort_of_dependencies_in_dependencymanagement)

**Comment by Ekryd** [comment]
I just discovered that the description of the scope order is incorrect. IMPORT is ranked first, I will fix that in master and update the 210 branch with the change

**Pull request merged**